pkg/pb/inspector.proto

```diff
@@ -47,6 +47,33 @@ service PieceStoreInspector {
   rpc Dashboard(DashboardRequest) returns (DashboardResponse) {}
 }
 
+service IrreparableInspector {
+  // ListSegments returns damaged segments
+  rpc ListSegments(ListSegmentsRequest) returns (ListSegmentsResponse);
+}
+
+// ListSegments
+message ListSegmentsRequest {
+  int64 limit = 1;
+  int64 offset = 2;
+}
+
+message IrreparableSegment {
+  bytes encrypted_path = 1;
+  bytes segment_detail = 2;
+  int64 lost_pieces = 3;
```

Comment:
I'm a fan of adding [(gogoproto.nullable) = false]; to things that will always have a value